### DISCUSSION TITLE
What is rustedanvil-core?

### DISCUSSION THREAD

**Discussion by Unloaded123456** [Abstrat-Technologies/rustedanvil#31] [General]
What is the rustedanvil-core repo? It looks like it is a fork.

**Comment by arek-kwapis** [comment]
The rustedanvil-core is indeed a fork - it's purpose is to give a stable development base that can be easily reused for the future. 

Say we finish 965 and move on to another forge version, we can easily use the rustedanvil-core code to have a ready-to-go base during it's milestone 2 development and so on.

**Reply by arek-kwapis** [reply]
To add to that - all the development for the next few weeks will happen on the core fork, until its ready, at which point we'll reverse-fork it into this repository

This will continue to be the main hub for issues, the wiki, discussions etc. Due to how forks work
